openfl/net/URLRequest.hx

```diff
@@ -13,7 +13,10 @@ import openfl.utils.ByteArray;
 	public var requestHeaders:Array<URLRequestHeader>;
 	public var url:String;
 	public var userAgent:String;
-	
+
+	#if (js && html5)
```

Comment:
Correct, ```js``` is not the same as ```html5```

Properly understood, ```js``` is similar to ```cpp``` whereas ```html5``` is similar to ```windows```. 

IE, ```html5``` implies `js`, but `js` does not imply `html5`. 

In practice you can have JS+HTML5 (ie, a browser app), or you can have JS+node (a non-browser-app).